forage-android/src/main/java/com/joinforage/forage/android/collect/AbstractVaultSubmitter.kt

```diff
@@ -0,0 +1,183 @@
+package com.joinforage.forage.android.collect
+
+import android.content.Context
+import com.joinforage.forage.android.VaultType
+import com.joinforage.forage.android.core.telemetry.Log
+import com.joinforage.forage.android.core.telemetry.UserAction
+import com.joinforage.forage.android.core.telemetry.VaultProxyResponseMonitor
+import com.joinforage.forage.android.model.EncryptionKeys
+import com.joinforage.forage.android.model.PaymentMethod
+import com.joinforage.forage.android.network.ForageConstants
+import com.joinforage.forage.android.network.model.ForageApiResponse
+import com.joinforage.forage.android.network.model.ForageError
+import com.joinforage.forage.android.network.model.UnknownErrorApiResponse
+import com.joinforage.forage.android.ui.ForagePINEditText
+import com.verygoodsecurity.vgscollect.core.HTTPMethod
+
+
+internal val IncompletePinError = ForageApiResponse.Failure.fromError(
+    ForageError(400, "user_error", "Invalid EBT Card PIN entered. Please enter your 4-digit PIN.")
+)
```

Comment:
I know we have a dupe inside the `ForageConstants`, but I figured it made more sense for it to live here - where it is only used.

The dupe will be removed once the legacy PIN collectors become obsolete